src/datasets/datasets.controller.ts

```diff
@@ -924,12 +924,13 @@ export class DatasetsController {
     description: "Id of the dataset to be modified",
     type: String,
   })
-  @ApiParam({
+  @ApiQuery({
     name: "fieldName",
     description: "Name of the field to be updated",
     type: String,
   })
-  @ApiBody({
+  @ApiQuery({
```

Comment:
Why have we changed from body to query param?
Are we sure that this endpoint is not used anywhere else?

Reply:
```
  public appendToArrayField(id: any, fieldName: any, data: any, customHeaders?: Function): Observable<any> {
    let _method: string = "POST";
    let _url: string = LoopBackConfig.getPath() + "/" + LoopBackConfig.getApiVersion() +
    "/Datasets/:id/appendToArrayField";
    let _routeParams: any = {
      id: id
    };
    let _postBody: any = {};
    let _urlParams: any = {};
    if (typeof fieldName !== 'undefined' && fieldName !== null) _urlParams.fieldName = fieldName;
    if (typeof data !== 'undefined' && data !== null) _urlParams.data = data;
    let result = this.request(_method, _url, _routeParams, _urlParams, _postBody, null, customHeaders);
    return result;
  }
```
I don't see this endpoint is used somewhere else yet
But even this endpoint is used somewhere else the body will always be empty object. You can see the frontend code: `let _postBody: any = {};`, its sending empty {} to backend by default and is not attached to any values.